sdk/attestation/azure-security-attestation/azure/security/attestation/_client.py

```diff
@@ -22,107 +17,151 @@
     InitTimeData,
     DataType,
     AttestSgxEnclaveRequest,
-    AttestOpenEnclaveRequest)
+    AttestOpenEnclaveRequest,
+)
 from ._configuration import AttestationClientConfiguration
-from ._models import (
-    AttestationSigner,
-    AttestationToken,
-    AttestationResponse,
-    AttestationResult,
-    AttestationData,
-    TpmAttestationRequest,
-    TpmAttestationResponse)
-import base64
-from azure.core.tracing.decorator import distributed_trace
-from threading import Lock
+from ._models import AttestationSigner, AttestationToken, AttestationResult
+from ._common import merge_validation_args
+
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from azure.core.credentials import TokenCredential
 
 
 class AttestationClient(object):
     """An AttestationClient object enables access to the Attestation family of APIs provided
       by the attestation service.
 
-    :param str instance_url: base url of the service
     :param credential: Credentials for the caller used to interact with the service.
     :type credential: :class:`~azure.core.credentials.TokenCredential`
-    :keyword pipeline: If omitted, the standard pipeline is used.
-    :paramtype pipeline: Pipeline
-    :keyword policies: If omitted, the standard pipeline is used.
-    :paramtype policies: list[HTTPPolicy]
-    :keyword HttpTransport transport: If omitted, the standard pipeline is used.
-    :paramtype transport: HttpTransport
+    :param str endpoint: The attestation instance base URI, for example https://mytenant.attest.azure.net.
```

Comment:
Now that `endpoint` comes before `credential` as a positional parameter, I'd list it first in the docstring for each Client class